clipboard/ios/Plugin/Clipboard.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Capacitor
+
+@objc public class Clipboard: NSObject {
+    
+    @objc enum ContentType: Int {
+        case string
+        case url
+        case image
+    }
+    
+    @objc func write(_ label: ContentType, _ content: String) {
+        switch label {
+        case ContentType.string:
+                UIPasteboard.general.string = content
+        case ContentType.url:
+            if let url = URL(string: content) {
+                UIPasteboard.general.url = url
+            }
+        case ContentType.image:
+            if let data = Data(base64Encoded: getCleanData(content)) {
+                let image = UIImage(data: data)
```

Comment:
These two steps should be combined into a single `if` statement since they are logically linked, could fail, and have a force unwrap below in the log statement (which will crash if the conversion did fail).
```suggestion
            if let data = Data(base64Encoded: getCleanData(content)), let image = UIImage(data: data) {
```